pkg/services/ngalert/prom/convert.go

```diff
@@ -211,10 +260,86 @@ func createAlertQueryNode(datasourceUID, datasourceType, expr string, fromTimeRa
 	return models.AlertQuery{
 		DatasourceUID: datasourceUID,
 		Model:         modelJSON,
-		RefID:         "A",
+		RefID:         refID,
 		RelativeTimeRange: models.RelativeTimeRange{
 			From: models.Duration(fromTimeRange + evaluationOffset),
 			To:   models.Duration(0 + evaluationOffset),
 		},
 	}, nil
 }
+
+func createMathNode() (models.AlertQuery, error) {
+	refID := "B"
```

Comment:
Maybe we should name the query node as "query", this one as "prometheus_math" and the threshold as "threshold"? To make it clear